# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1295303142 18000
#      Mon Jan 17 17:25:42 2011 -0500
# Node ID 1140bd9f9b213f76f801ffc6085fe57ed4346a6a
# Parent  05b5bcdb09c8151f362198c0d1efb1de350c9db0
mkoctfile.cc.in: add + missing from previous change

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,12 @@
+2011-01-17  John W. Eaton  <jwe@octave.org>
+
+	* mkoctfile.cc.in (main): Add + missing from previous change.
+
 2011-01-15  John W. Eaton  <jwe@octave.org>
 
 	* configure.ac (AC_INIT): Version is now 3.3.54.
 	(OCTAVE_API_VERSION_NUMBER): Now 42.
 	(OCTAVE_RELEASE_DATE): Now 2010-11-19.
 
 2011-01-15  John W. Eaton  <jwe@octave.org>
 
diff --git a/mkoctfile.cc.in b/mkoctfile.cc.in
--- a/mkoctfile.cc.in
+++ b/mkoctfile.cc.in
@@ -739,17 +739,17 @@ main (int argc, char **argv)
 	  if (!vars["LD_CXX"].empty ())
 	    {
 	      string cmd = vars["LD_CXX"] + " " + vars["CPPFLAGS"] + " "
 		+ vars["ALL_CXXFLAGS"] + " " + vars["RDYNAMIC_FLAG"]
 		+ " " + vars["ALL_LDFLAGS"] + " " +  pass_on_options
 		+ " " + output_option + " " + objfiles + " " + libfiles
 		+ " " + ldflags + " " + vars["LFLAGS"] + " "
 		+ vars["RLD_FLAG"] + " " + vars["OCTAVE_LIBS"] + " "
-		+ vars["LAPACK_LIBS"] + " " vars["BLAS_LIBS"] + " "
+		+ vars["LAPACK_LIBS"] + " " + vars["BLAS_LIBS"] + " "
                 + vars["FFTW_LIBS"] + " " + vars["READLINE_LIBS"] + " "
                 + vars["LIBS"] + " " + vars["FLIBS"];
 	      result = run_command (cmd);
 	    }
 	  else
 	    {
 	      cerr << "mkoctfile: no way to link stand-alone executable file"
 		   << endl;
